### PR TITLE
Above-the-fold copy updates + SEO/ performance improvements

### PR DESCRIPTION
**Pull request by aeolianeth**
Discord convo for context: https://discord.com/channels/775859454780244028/968886189157412864

The primary intention of this PR is to improve clarity and "snapiness" of the copy used on the home page above-the-fold. By clearly stating what Juicebox is (with a bit of a flex), the goal is for people to have a clearer idea of what Juicebox is from their first interaction with the site.

- updates homepage hero heading and byline
- renames CTA button to `Launch your project`
- Adds a secondary CTA `Explore projects`

I also got carried away with some SEO/perf related changes:
- resizes all images to match the size their actually used at.
- imageoptim all images
- a few other things here and there.

| before | after |
| --- | --- |
| Screen Shot 2022-05-13 at 9 58 57 pm | Screen Shot 2022-05-13 at 9 59 58 pm |
| Screen Shot 2022-05-13 at 10 24 45 pm | Screen Shot 2022-05-13 at 10 24 27 pm |

### Lighthouse

Note: performance is probablt better in the `before` because this is testing juicebox.money vs the preview link - the preview link tends to be quite a bit slower.

| before | after |
| --- | --- |
| Screen Shot 2022-05-13 at 10 22 21 pm | Screen Shot 2022-05-13 at 10 20 43 pm |

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <meta property="og:title" content="Juicebox" />
     <meta
       property="og:description"
-      content="Community funding for people and projects on Ethereum."
+      content="The world's largest decentralized funding platform. Built on Ethereum."
```
Comment (aeolianeth) [inline public/index.html:11]: juicier meta

File: src/components/Landing/index.tsx [hunk 2]
```diff
@@ -94,112 +171,42 @@ export default function Landing() {
...
                       }}
-                      href="https://ethereum.org/en/what-is-ethereum/"
                     >
-                      Ethereum
-                    </ExternalLink>
-                    .
-                  </Trans>
-                </div>
+                      <Trans>
+                        Juicebox is the world's largest decentralized funding
```
Comment (aeolianeth) [inline src/components/Landing/index.tsx:190]: New byline

File: public/index.html (hunk 1 above)
Comment (wraeth-eth) [reply public/index.html:15]: 😎

File: src/components/Landing/Footer.tsx [hunk 3]
```diff
@@ -30,9 +32,9 @@ export default function Footer() {
 
   // Renders language links
   const languageLink = (lang: string) => (
-    <span key={lang} onClick={() => setLanguage(lang)}>
-      {link(Languages[lang].long)}
-    </span>
+    <Button key={lang} onClick={() => setLanguage(lang)} type="link">
```
Comment (aeolianeth) [inline src/components/Landing/Footer.tsx:35]: These aren't links, they're buttons - change accordingly

File: src/components/Landing/index.tsx [hunk 4]
```diff
@@ -422,9 +446,10 @@ export default function Landing() {
 
             <Col xs={24} md={10}>
               <img
-                style={{ maxWidth: '100%' }}
+                style={{ maxWidth: 420 }}
```
Comment (aeolianeth) [inline src/components/Landing/index.tsx:449]: :420:

File: src/components/shared/RichImgPreview.tsx [hunk 5]
```diff
@@ -35,6 +36,7 @@ export default function RichImgPreview({
             width: w,
             height: h,
           }}
+          alt={t`Payment memo image`}
```
Comment (aeolianeth) [inline src/components/shared/RichImgPreview.tsx:39]: missing alt

File: src/styles/index.scss [hunk 6]
```diff
@@ -125,7 +125,7 @@ button {
 }
 
 body, .top-nav, div {
-  transition: background-color 750ms;
+  transition: background-color 250ms linear;
```
Comment (aeolianeth) [inline src/styles/index.scss:128]: speed it up a touch

**Comment by aeolianeth** [comment]
@peripheralist definitely would like your review of this one if possible 🙏

**Comment by aeolianeth** [comment]
@johnnyd-eth could you please review this PR?

⚠️ please don't merge yet. I'd like to get @peripheralist's ✅  on the new copy before proceeding. Cheers

**Pull request merged**